src/modules/Dashboard/features/Managers/Managers.utils.tsx

```diff
@@ -8,33 +8,13 @@ import { variables } from 'shared/styles';
 
 import { ManagersActions } from './Managers.types';
 
-export enum ManagersColumnsWidth {
-  Pin = '4.8rem',
-  Default = '22rem',
-  Email = '35rem',
-}
-
 export const getHeadCells = (id?: string): HeadCell[] => {
   const { t } = i18n;
 
   return [
     {
       id: 'checkbox',
-      label: (
-        <Checkbox
-          aria-label="Check all"
-          checked={false}
-          data-testid="dashboard-managers-checkbox-header"
-          icon={
-            <Svg
-              fill={variables.palette.outline_variant2}
-              height="20"
-              id="checkbox-empty-outlined"
-              width="20"
-            />
-          }
-        />
-      ),
+      label: <Checkbox aria-label={t('checkAll')} checked={false} />,
```

Comment:
Yeah, I copied it over from the existing similar column in `Participants`, and just renamed it initially.

But this isn't used for any tests yet, because no bulk operation functionality has been implemented yet. I figured I'd remove unused attributes to reduce potential confusion, and let whoever ends up responsible for adding this feature take ownership and add it if the need it (who knows if this Checkbox will exist as-is when it comes to actually building the feature)?